src/pages/en/guides/deploy/kinsta.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Deploy your Astro Site to Kinsta Application Hosting
+description: How to deploy your Astro site to the web on Kinsta Application Hosting.
+layout: ~/layouts/DeployGuideLayout.astro
+i18nReady: true
+---
+
+You can use [Kinsta Application Hosting](https://kinsta.com/application-hosting/) to host an Astro site on their cloud hosting.
+
+:::tip[Looking for an example?]
+Check out [the official Kinsta Application Hosting Starter project for Astro](https://github.com/kinsta/hello-world-astro)!
+:::
+
+## Project configuration
+To host your project on **Kinsta Application Hosting**, you need to:
+- make sure your `package.json` has a `name` field.
```

Comment:
Can you describe what should/shouldn't go here? I notice that in your example it's not formatted like a domain, and it starts with an `@` and also seems to have a subdirectory. I can't tell from this what the proper format has to be, and whether these things are all requirements. Can you provide further guidance on this for the reader?

Reply:
For some reason, Buildpacks (the tech behind building everything on Kinsta) requires to have the "name" field. It's required and that's it. You could add any text there.

It's quite important because sometimes there are package.json files without this field.

Reply:
interesting! So, it's "used" somehow by Buildpacks, but not in any way that would affect your projet? It could literally be *anything*?

Reply:
You can enter "qwerty," and it will be OK. It's weird, but it is as it is.

Reply:
I'll take it! Just need to know how to describe it. lol... not my place to judge... 😄

Reply:
I think the later commit about the package name solves everything, and we can close this.